src/pyOpenMS/pyopenms/_dataframes.py

```
@@ -657,78 +706,34 @@ def _add_meta_values(df: _pd.DataFrame, object: any) -> _pd.DataFrame:
+            elif dv.valueType() == _DataValue.DOUBLE_VALUE:
+                value = dv.toDouble()
+                dtype = "float64"
+            elif dv.valueType() == _DataValue.EMPTY_VALUE:
+                continue
+            else:
+                value = str(dv)
+                dtype = "object"
+            
+            df[col_name] = _np.full(df.shape[0], value, dtype=dtype)
+            
+        except Exception: 
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)